api/apps/api/src/modules/scenarios/planning-units/scenario-planning-units.service.ts

```diff
@@ -28,6 +28,18 @@ export class ScenarioPlanningUnitsService {
     });
   }
 
+  async getByStatus(
+    scenarioId: string,
+    lockStatus: LockStatus,
+  ): Promise<ScenariosPlanningUnitGeoEntity[]> {
+    return this.puRepo.find({
```

Comment:
I usually recommend to use `return await` _always_ even when not strictly needed, mostly for the reasons put forward in this comment: https://stackoverflow.com/questions/38708550/difference-between-return-await-promise-and-return-promise#comment128897650_70198397

In practice, linters should take care of guiding us and letting us not have to keep all these details in mind. As long as the reasons for using either arrangement are clear, I have no strong opinions though.